opae.spec

```diff
@@ -58,13 +60,15 @@ OPAE headers, tools, sample source, and documentation
 
 
 %{?python_disable_dependency_generator}
+# Workaround a problem with pybind11 *.so not having build-id's
+%undefine _missing_build_ids_terminate_build
 
 
 %prep
 %setup -q -n %{name}-%{version}-%{opae_release}
 
 %build
-%cmake -DCMAKE_INSTALL_PREFIX=/usr  -DOPAE_PRESERVE_REPOS=ON -DOPAE_BUILD_LEGACY=ON -DOPAE_BUILD_SAMPLES=ON -DOPAE_BUILD_EXTRA_TOOLS_FPGABIST=ON .
```

Comment:
Better leave `-DOPAE_BUILD_SAMPLES=ON ` in here, and remove the manual installation of sample sources below?